mw-webapp/src/dataAccessLogic/renderCellValue/helpers/updateCell.ts

```diff
@@ -0,0 +1,47 @@
+import {updateCurrentProblem} from "src/dataAccessLogic/renderCellValue/helpers/updateCurrentProblem";
+import {updateDayReport} from "src/dataAccessLogic/renderCellValue/helpers/updateDayReport";
+import {updateJobDone} from "src/dataAccessLogic/renderCellValue/helpers/updateJobDone";
+import {updateMentorComment} from "src/dataAccessLogic/renderCellValue/helpers/updateMentorComment";
+import {updatePlanForNextPeriod} from "src/dataAccessLogic/renderCellValue/helpers/updatePlanForNextPeriod";
+import {ColumnNameProps} from "src/dataAccessLogic/renderCellValue/renderCellItem";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+
+const updateCellsFunctions: Record<string, (text: string, uuid: string) => Promise<void>> = {
+  updateJobDone,
+  updatePlanForNextPeriod,
+  updateCurrentProblem,
+  updateMentorComment,
+};
+
+/**
+ * Update cells
+ */
+const updateCells = (name: string, text: string, uuid: string) => {
```

Comment:
does cells have common name? HOw does iit possible? why do we need cell name? Check signatures everywhere

Reply:
changed to nameOfFunction. It's a generic for call different function depnd on column name